game/views.py

```
@@ -643,50 +643,53 @@ def ai_move(request):
+                    # Cap the alts before running the notation loop
+                    for alt in opp_resp.get('alts', [])[:2]:
+                        predicted_responses.append({
+                            'notation': temp_game._notation(
+                                alt['from_row'], alt['from_col'],
+                                alt['to_row'], alt['to_col'],
+                                temp_game.board[alt['from_row']][alt['from_col']],
+                                temp_game.board[alt['to_row']][alt['to_col']],
+                                temp_game.serialize_board(), temp_game.serialize_castling_rights(), temp_game._serialize_ep()
+                            ),
+                            'eval': alt.get('eval')
+                        })
```

Ruff (0.15.21)

[warning] Use `list.extend` to create a transformed list

(PERF401)